pkgs/top-level/all-packages.nix

```diff
@@ -9865,7 +9865,7 @@ in
   llvm_6  = llvmPackages_6.llvm;
   llvm_5  = llvmPackages_5.llvm;
 
-  llvmPackages = recurseIntoAttrs llvmPackages_7;
+  llvmPackages = recurseIntoAttrs llvmPackages_8;
```

Comment:
Given that LLVM 11 is out, why are we so conservative here?

Reply:
Ask @NixOS/darwin-maintainers I would assume people try to bump this all the time and there are issues.